test/squish/deadline_cloud_samples/blender_render/template.yaml

```diff
@@ -0,0 +1,105 @@
+specificationVersion: 'jobtemplate-2023-09'
```

Comment:
Can / Should this be in a different repository?

Reply:
I thought about it but the tests need to be able to pull/use these files when running and AFAIK it would have to live in the same repository as the code itself. 

Note that I am pulling test files from [deadline-cloud-samples](url) repo -- I am mainly doing this because I don't know if there is some standard for test file creation that we would want to offer our users (and I am also worried about open source contributors modifying existing test files causing security issues -- as mentioned in Cody's doc), and everything in deadline-cloud-samples is already repo and I assume vetted already.

Reply:
Yeah, and this is why I'm asking because we now have blender related test assets / files in different places. Would a DCC agnostic test make more sense? I am not sure how we can make it "generic"

Reply:
I'm not sure - I know the Submitter dialogue displays different options (such as conda package versions) for each different DCC (so that was my reasoning for automating the different dialogues). Let me think if there is a way to do DCC agnostic tests -- My worry would be if there is a loss of test coverage here.